src/uct/sm/self/self_md.c

```diff
@@ -9,7 +9,8 @@
 static ucs_status_t uct_self_md_query(uct_md_h md, uct_md_attr_t *attr)
 {
     /* Dummy memory registration provided. No real memory handling exists */
-    attr->cap.flags         = UCT_MD_FLAG_REG;
+    attr->cap.flags         = UCT_MD_FLAG_REG |
```

Comment:
will it work with zero packed size?

Reply:
yes, even though rkey pack should not be called on this md in the first place